sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverClient.java

```diff
@@ -260,20 +244,6 @@ public ServiceBusReceivedMessage peekMessageAt(long sequenceNumber) {
         return asyncClient.peekMessageAt(sequenceNumber).block(operationTimeout);
     }
 
-    /**
-     * Starting from the given sequence number, reads next the active message without changing the state of the receiver
-     * or the message source.
-     *
-     * @param sequenceNumber The sequence number from where to read the message.
-     * @param sessionId Session id of the message to peek from. {@code null} if there is no session.
-     *
-     * @return A peeked {@link ServiceBusReceivedMessage}.
-     * @see <a href="https://docs.microsoft.com/azure/service-bus-messaging/message-browsing">Message browsing</a>
-     */
-    public ServiceBusReceivedMessage peekMessageAt(long sequenceNumber, String sessionId) {
```

Comment:
For the async client, we just hid these overloads by making them package-private. In sync client, we are deleting these overloads. We should be consistent by either deleting in both clients or hiding it in both clients.